uvtools/plot.py

```diff
@@ -723,3 +723,185 @@ def plot_diff_uv(uvd1, uvd2, pol=None, check_metadata=True, bins=50):
         fig.colorbar(cax)
 
     return fig
+
+def plot_diff_1d(uvd1, uvd2, antpairpol, plot_type="both", 
+                 check_metadata=True, dimension=None):
+    # TODO: docstring
+    """Produce plots of visibility differences along a single axis.
```

Comment:
Would be good to add a note on how the averaging is done (i.e. if it takes into account weights, flags, or similar).

Reply:
I've updated the description in the `Notes` section. With the update mentioned below, weighted averages are now allowed (with user-specified weights), but flags are not taken into account. I can make an issue to add support for flagged arrays if you think it's an important enough feature to add.

Reply:
Something worth noting is that the addition of the `**kwargs` parameter allows users to actually retrieve the averaged visibility array by specifying the `out` parameter (although `np.average` does *not* support this).